docs/sql-ref-syntax-dml-dynamic-partition-insert.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: global
+title: Dynamic Partition Inserts
+displayTitle: Dynamic Partition Inserts
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+When the partition value is not provided, such inserts are called as the dynamic partition inserts, also called as multi-partition inserts. In partition spec, the partition column values are optional. When the values are not given, these columns are referred to as dynamic partition columns; otherwise, they are static partition columns. For example, the partition spec (p1 = 3, p2, p3) has a static partition column (p1) and two dynamic partition columns (p2 and p3).
```

Comment:
This looks like it's copied from the Databricks documentation: https://docs.databricks.com/spark/latest/spark-sql/language-manual/insert.html In general we can't just copy content from third party sources.

I think we'd have to check whether this is OK to contribute as OSS docs, and it might be. However there are references below to Databricks runtime that would not be appropriate or relevant.

Before going further, were any other of the sections here or in other PRs copied from elsewhere?

Reply:
I refer to Databricks documentation but only directly copied this ```dynamic partition insert ```because I am not familiar with this part. I will check and rewrite.

Reply:
@srowen I removed dynamic partition insert. I also looked closely to make sure other insert files are OK. Could you please review one more time?